netbox/templates/ipam/vlantranslationpolicy.html

```diff
@@ -0,0 +1,37 @@
+{% extends 'generic/object.html' %}
+{% load helpers %}
+{% load plugins %}
+{% load render_table from django_tables2 %}
+{% load i18n %}
+
+{% block content %}
+<div class="row">
```

Comment:
Indentation is not consistent; should be two spaces for every child element/tag.